apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktDao.kt

```diff
@@ -61,4 +67,93 @@ class AktivitetspliktDao(private val connectionAutoclosing: ConnectionAutoclosin
             }
         }
     }
+
+    fun hentAktiviteter(behandlingId: UUID): List<AktivitetspliktAktivitet> {
+        return connectionAutoclosing.hentConnection {
+            with(it) {
+                val stmt =
+                    prepareStatement(
+                        """
+                        SELECT id, sak_id, behandling_id, aktivitet_type, fom, tom, opprettet, endret, beskrivelse
+                        FROM aktivitetsplikt_aktivitet
+                        WHERE behandling_id = ?
+                        """.trimMargin(),
+                    )
+                stmt.setObject(1, behandlingId)
+
+                stmt.executeQuery().toList { toAktivitet() }
+            }
+        }
+    }
+
+    fun opprettAktivitet(
+        behandlingId: UUID,
+        aktivitet: OpprettAktivitetspliktAktivitet,
+        kilde: Grunnlagsopplysning.Kilde,
+    ) {
+        return connectionAutoclosing.hentConnection {
+            with(it) {
+                val stmt =
+                    prepareStatement(
+                        """
+                        INSERT INTO aktivitetsplikt_aktivitet(id, sak_id, behandling_id, aktivitet_type, fom, tom, opprettet, endret, beskrivelse) 
+                        VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)
+                        """.trimMargin(),
+                    )
+                stmt.setObject(1, UUID.randomUUID())
+                stmt.setLong(2, aktivitet.sakId)
+                stmt.setObject(3, behandlingId)
+                stmt.setString(4, aktivitet.type.name)
+                stmt.setDate(5, Date.valueOf(aktivitet.fom))
+                stmt.setDate(6, aktivitet.tom?.let { tom -> Date.valueOf(tom) })
+                stmt.setString(7, kilde.toJson())
+                stmt.setString(8, kilde.toJson())
+                stmt.setString(9, aktivitet.beskrivelse)
+
+                stmt.executeUpdate()
+            }
+        }
+    }
+
+    private fun ResultSet.toAktivitet() =
+        AktivitetspliktAktivitet(
+            id = getUUID("id"),
+            sakId = getLong("sak_id"),
+            behandlingId = getUUID("behandling_id"),
+            type = AktivitetspliktAktivitetType.valueOf(getString("aktivitet_type")),
+            fom = getDate("fom").toLocalDate(),
+            tom = getDate("tom")?.toLocalDate(),
+            opprettet = objectMapper.readValue(getString("opprettet")),
+            endret = objectMapper.readValue(getString("endret")),
+            beskrivelse = getString("beskrivelse"),
+        )
+}
+
+data class AktivitetspliktAktivitet(
+    val id: UUID,
+    val sakId: Long,
+    val behandlingId: UUID,
+    val type: AktivitetspliktAktivitetType,
+    val fom: LocalDate,
+    val tom: LocalDate?,
+    val opprettet: Grunnlagsopplysning.Kilde,
+    val endret: Grunnlagsopplysning.Kilde?,
+    val beskrivelse: String,
+)
+
+data class OpprettAktivitetspliktAktivitet(
+    val id: UUID? = null,
```

Comment:
Trenger denne en id?

Reply:
Ikke egentlig akkurat nå, men kommer en ny PR etterhvert hvor jeg ser for meg at samme objektet kommer til å bli brukt for å oppdatere også.